data/migrations/20200124094001_create-all-tables.js

```diff
@@ -0,0 +1,55 @@
+exports.up = function(knex) {
+  return knex.schema
+    .createTable("projects", table => {
+      table.increments();
+      table.string("name", 128).notNullable();
+      table.string("description");
+      table.boolean("completed").defaultTo(false);
+    })
+    .createTable("tasks", table => {
+      table.increments();
+      table
+        .integer("project_id")
+        .unsigned()
+        .notNullable()
+        .references("id")
+        .inTable("projects")
+        .onUpdate("CASCADE")
+        .onDelete("CASCADE");
+      table.integer("step");
```

Comment:
Is `step`  an attribute of `tasks` from the requirement?

Reply:
No, I added it on my own because I thought it made sense.